### ISSUE TITLE
[Feature Request] Would it be possible to implement the "VPROJECT=steammulti" and "start "Steam1" "-master_ipc_name_override steam1" on this for multiple steam logins?

### ISSUE THREAD

**Issue by Kevauz** [TCNOco/TcNo-Acc-Switcher#406]
Would be a game changer for multi account users.

**Comment by TCNOco** [comment]
I assume you mean something similar to this? https://www.reddit.com/r/fo76/comments/1ajs0qm/guide_on_how_to_run_multiple_steam_accounts_on/

I assume you can do this with the built-in account switcher on Steam, as for this it just changes the last logged in user. I'll have to look into this to see if something is possible.

**Comment by Kevauz** [comment]
> I assume you mean something similar to this? https://www.reddit.com/r/fo76/comments/1ajs0qm/guide_on_how_to_run_multiple_steam_accounts_on/
> 
> I assume you can do this with the built-in account switcher on Steam, as for this it just changes the last logged in user. I'll have to look into this to see if something is possible.

Yep, exactly used that. My only qualm with it is that I have 6 accounts and steam only saves the last 5 so it takes awhile to swap (my use case right now is using 4 at the same time)/

**Comment by TCNOco** [comment]
As far as I understand this is patched on Steam's side now and does not make sense to implement. Please let me know if I'm wrong!

The app has changed a lot over the years and underwent a major rewrite. Check out the latest releases under Pre-release:
https://github.com/TCNOco/TcNo-Acc-Switcher/releases
